primitives/src/lib.rs

```diff
@@ -56,15 +65,20 @@ pub type Hash = sp_core::H256;
 /// time scale is milliseconds.
 pub type Timestamp = u64;
 
+pub type CurrencyId = u32;
```

Comment:
Why do we need this? Since we do not support multiple currencies

Reply:
:-) Clean-up done :+1: